packages/lib/src/createUniqueClassName.ts

```diff
@@ -0,0 +1,25 @@
+// https://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid
+// This is a RFC4122 version 4 compliant solution that solves that issue by offsetting the first 13 hex numbers by a hex portion of the timestamp, and once depleted offsets by a hex portion of the microseconds since pageload.
+// That way, even if Math.random is on the same seed, both clients would have to generate the UUID the exact same number of microseconds since pageload (if high-perfomance time is supported) AND at the exact same millisecond (or 10,000+ years later) to get the same UUID:
+
+const generateUUID = () => {
+  let date1 = new Date().getTime(),
+    date2 = (performance && performance.now && performance.now() * 1000) || 0;
+  return 'xxxxxxxx'.replace(/[xy]/g, char => {
+    let rand = Math.random() * 16;
+    if (date1 > 0) {
+      rand = (date1 + rand) % 16 | 0;
+      date1 = Math.floor(date1 / 16);
+    } else {
+      rand = (date2 + rand) % 16 | 0;
+      date2 = Math.floor(date2 / 16);
+    }
```

Comment:
If we're only using 8 chars, this block will never get called, since `Date.getTime / (16^8)` will always be `> 0`. (Doesn't look like it'll roll over until the 11th char). So in that case, we won't even really need to worry about `performance`